openapi-diff/src/modeler/AutoRest.Swagger.Tests/SwaggerModelerCompareTests.cs

```diff
@@ -888,10 +888,48 @@ public void XmsEnumModelAsString()
         [Fact]
         public void ChangedParameterOrder()
```

Comment:
@mikekistler can you please have a look at all the 3 tests here and let me know if this is the behavior you desire?

Currently the test `ChangedParameterOrder` passes but the remaining two tests fail. If you confirm the tests are correct, I will adjust the implementation to make the tests pass.

Additional context here:
- https://github.com/Azure/azure-sdk-tools/issues/7170#issuecomment-2162156876

Reply:
I can only see two tests added and both of these look right to me.